opencti-platform/opencti-graphql/src/modules/internal/csvMapper/csvMapper-domain.ts

```diff
@@ -100,7 +101,7 @@ export const csvMapperSchemaAttributes = async (context: AuthContext, user: Auth
   types.forEach((type) => {
     const attributesDef = schemaAttributesDefinition.getAttributes(type);
     const attributes: CsvMapperSchemaAttribute[] = Array.from(attributesDef.values()).flatMap((attribute) => {
-      if (INTERNAL_ATTRIBUTES.includes(attribute.name)) return [];
+      if (INTERNAL_ATTRIBUTES.includes(attribute.name) || attribute === created) return [];
```

Comment:
I still don't like this solution. Why do we ignore this field created ? it's an information that could be useful for all entities and could be present in a CSV file or feed, no ? @nino-filigran